host/sgx/create.c

```diff
@@ -363,6 +368,12 @@ static oe_result_t _calculate_enclave_size(
         image_size + heap_size +
         (size_settings->num_tcs * (stack_size + tls_size + control_size));
 
+    /* Check if 0-base enclave */
+    if (props->config.flags.create_zero_base_enclave)
+    {
+        *loaded_enclave_pages_size += props->config.start_address;
```

Comment:
Adjusting the loaded_enclave_pages_size here also changes the size returned. Might impact simulation mode. See the other comment in later section.

Reply:
Added check to not increase size, if in simulation mode.

Reply:
About the question if enclave_elrange.elrange_size overwrites SECS.size - Yes. The SECS.size expected during measurements is the elrange_size.